tabulardelta/formatters/detailed_text_formatter.py

```diff
@@ -105,7 +112,13 @@ def format(self, delta: TabularDelta) -> str:
 
         if self.value_changes > 0:
             for chg in delta.cols.differences:
-                result.append([_show_values(chg, self.value_changes + 3, True)])
+                result.append(
+                    [
+                        _show_values(
+                            chg, self.value_changes + 3, self.value_change_height, True
+                        )
+                    ]
+                )
```

Comment:
I maybe am missing something here but is it possible to have _show_values(..) return list[cell] such that you don't need to wrap the result in another indentation (which just makes it slightly harder to read?)